src/data/Storage.ts

```diff
@@ -54,133 +54,147 @@ export const storage = {
 
 // NOTE: Predefined templates for test purpose
 //       for use just uncomment one of the following lines:
-// _emptyArrayOfCycles().catch((err) => console.error(err));
-// _arrayOfCyclesWithSingleCycle().catch((err) => console.error(err));
-// _arrayOfCyclesWithSingleCycleWithDelay().catch((err) => console.error(err));
-// _halfFilledArrayOfCycles().catch((err) => console.error(err));
-// _fullyFilledArrayOfCycles().catch((err) => console.error(err));
-// _fullyFilledArrayOfCyclesWithDelay().catch((err) => console.error(err));
+
+// _todayPeriod(6).catch((err) => console.error(err));
+// _todayOvulation(6).catch((err) => console.error(err));
+// _tomorrowOvulation(6).catch((err) => console.error(err));
+// _menstrualPhase(6).catch((err) => console.error(err));
+// _follicularPhase(6).catch((err) => console.error(err));
+// _lutealPhase(6).catch((err) => console.error(err));
+// _delayOfCycle(6).catch((err) => console.error(err));
 
 function _emptyArrayOfCycles(): Promise<void> {
   return storageImpl.remove("cycles") as Promise<void>;
 }
 
-function _arrayOfCyclesWithSingleCycle(): Promise<void> {
-  return storage.set.cycles([
-    {
+function _todayPeriod(countOfCycles: number): Promise<void> {
```

Comment:
Do we really needed in `countOfCycles` for a simple template functions which we use just for functionality check?

Reply:
This is an important parameter. So we can see on the "Details" what a different number of cycles will look like.

Reply:
Yes, but we already have a function which generates empty array. Maybe it's better to modify this function and let other functions just create cycles which we waiting from them?
For example, from ovulationToday function we waiting for situation when ovulation is today and it's doesnt matter how much cycles it generates. Same for another functions. But if we introduce function which generates required number of cycles that will be more clear in my opinion.